build.cmd

```diff
@@ -6,10 +6,28 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly.
 
+:: If neither managed nor native are passed as arguments, default to building both
+set __buildSpec=both
+
+:Arg_Loop
+:: Read in the args to determine whether to run the native build, managed build, or both
+if [%1] == [] goto :Tools
```

Comment:
I don't really like this hacking. I think we should just detect native or managed as the first argument and shift it. We don't need to try and stick with the msbuild syntax.